WalletWasabi/WabiSabi/Crypto/RoundHasher.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using NBitcoin;
+using WalletWasabi.Crypto;
+using WalletWasabi.Crypto.StrobeProtocol;
+
+namespace WalletWasabi.WabiSabi.Crypto
+{
+	public static class RoundHasher
+	{
+		public static uint256 CalculateHash(
+				TimeSpan inputRegistrationTimeout,
+				TimeSpan connectionConfirmationTimeout,
+				TimeSpan outputRegistrationTimeout,
+				TimeSpan transactionSigningTimeout,
+				long minRegistrableAmount,
+				long maxRegistrableAmount,
+				long maxRegistrableVsize,
+				long maxVsizeAllocationPerAlice,
+				CredentialIssuerParameters amountCredentialIssuerParameters,
+				CredentialIssuerParameters vsizeCredentialIssuerParameters,
+				long feePerK)
+				=> StrobeHasher.Create(ProtocolConstants.RoundStrobeDomain)
+					.Append(ProtocolConstants.RoundInputRegistrationTimeoutStrobeLabel, inputRegistrationTimeout)
+					.Append(ProtocolConstants.RoundConnectionConfirmationTimeoutStrobeLabel, connectionConfirmationTimeout)
+					.Append(ProtocolConstants.RoundOutputRegistrationTimeoutStrobeLabel, outputRegistrationTimeout)
+					.Append(ProtocolConstants.RoundTransactionSigningTimeoutStrobeLabel, transactionSigningTimeout)
+					.Append(ProtocolConstants.RoundMinRegistrableAmountStrobeLabel, minRegistrableAmount)
+					.Append(ProtocolConstants.RoundMaxRegistrableAmountStrobeLabel, maxRegistrableAmount)
```

Comment:
should use MPTX parameters, which are money ranges

Reply:
more generally everything in MPTX params should be covered if it's not, since that's a subset of the round params that concerns with the tx construction rules, those are still round params.

Reply:
Reposting my clarifications from slack for completeness:

Remove `MinRegistrableAmount`, `MaxRegistrableAmount`, replace with `MultipartyTransactionParameters` properties `AllowedInputAmounts` and `AllowedOutputAmounts` which are pairs of values.

Also add the following from `MultipartyTransactionParameters`:

- `AllowedInputTypes`
- `AllowedOutputTypes`
- `Network`
- `MaxTransactionSize`
- `MinRelayTxFee` (basically a constant, so maybe not necessary, but if core ever changes standardness policy this value matters)